kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -3365,6 +3363,33 @@ Styling a context menu for kview ui/ux controls
     -fx-background-color: -fx-focused-mark-color;
     -fx-alignment: center-right;
 }
+
+.kview-context-menu .menu-item .comment-icon{
+    -fx-scale-shape: false;
+    -fx-background-color: white;
+    -fx-shape: "M7.50002 18.3337C7.27901 18.3337 7.06705 18.2459 6.91077 18.0896C6.75448 17.9333 6.66669 17.7213 6.66669 17.5003V15.0003H3.33335C2.89133 15.0003 2.4674 14.8247 2.15484 14.5122C1.84228 14.1996 1.66669 13.7757 1.66669 13.3337V3.33366C1.66669 2.40866 2.41669 1.66699 3.33335 1.66699H16.6667C17.1087 1.66699 17.5326 1.84259 17.8452 2.15515C18.1578 2.46771 18.3334 2.89163 18.3334 3.33366V13.3337C18.3334 13.7757 18.1578 14.1996 17.8452 14.5122C17.5326 14.8247 17.1087 15.0003 16.6667 15.0003H11.5834L8.50002 18.092C8.33335 18.2503 8.12502 18.3337 7.91669 18.3337H7.50002ZM8.33335 13.3337V15.9003L10.9 13.3337H16.6667V3.33366H3.33335V13.3337H8.33335Z";
+}
```

Comment:
Can we use the existing comment icon? Is there one already created?

Reply:
Same goes for the other ones below: remove, edit, favorites, etc.

Reply:
Reused the icons.